src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -19,11 +20,17 @@ public Movie add(Movie movie) {
 
     @Override
     public Movie get(Long id) {
-        return movieDao.get(id).get();
+        if (id == null || id < 0) {
+            throw new IllegalArgumentException("Invalid cinema hall ID: " + id);
```

Comment:
The exception message refers to 'cinema hall ID' instead of 'movie ID'. This should be corrected to accurately reflect the context.